gammapy/__init__.py

```diff
@@ -74,3 +77,37 @@ def song(karaoke=False):
 
         centered = "\n".join(f"{s:^80}" for s in lyrics.split("\n"))
         sys.stdout.write(centered)
+
+
+# Set the bibtex entry to the article referenced in CITATION.
+def _get_bibtex():
+    try:
+        refs = (
+            (Path(__file__).parent.parent / "CITATION")
+            .read_text()
+            .split("@article")[1:]
```

Comment:
If you surround the blessed text snippets with triple quotes you could split on that instead and just use a fixed index to access the text without any need for clean up, and the file stays just as human readable as before.